src/librustc_trans/trans/common.rs

```diff
@@ -547,7 +552,7 @@ impl<'a, 'tcx> FunctionContext<'a, 'tcx> {
 // code.  Each basic block we generate is attached to a function, typically
 // with many basic blocks per function.  All the basic blocks attached to a
 // function are organized as a directed graph.
-pub struct BlockS<'blk, 'tcx: 'blk> {
+pub struct BlockS {
```

Comment:
It's pretty bad that we export both Block and BlockS types from this module. Previously we only used Block and it was a typedef. If we need to use both publicly, then we should change BlockS to have a more descriptive name - BlockGraph or BlockSet or something.